shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -118,22 +135,19 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         if (virtualViewId == View.NO_ID) {
             AccessibilityNodeInfo result = AccessibilityNodeInfo.obtain(mOwner);
             mOwner.onInitializeAccessibilityNodeInfo(result);
-            if (mObjects.containsKey(ROOT_NODE_ID))
-                result.addChild(mOwner, ROOT_NODE_ID);
+            if (mObjects.containsKey(ROOT_NODE_ID)) result.addChild(mOwner, ROOT_NODE_ID);
```

Comment:
To be consistent with our other java code, I'd keep the line break after the if condition. Maybe add { } if it makes the formatter happy.

Here and all other ifs below.

Reply:
Added back most of the {}, that seems to make clang format happy